release-notes/6.0/linux-packages.md

```diff
@@ -0,0 +1,99 @@
+# Linux package dependencies
+
+.NET 6 has several dependencies that must be satisfied to run .NET apps. The commands to install these libraries are listed for multiple distributions.
```

Comment:
There's overloading of the word "distribution" in these changes that can be ambiguous in some cases.  I would suggest using "Linux distribution" in cases like this.